presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCachingFileSystem.java

```diff
@@ -93,11 +87,47 @@ public void setupMethod()
     @Test(timeOut = 30_000)
     public void testBasic()
             throws Exception
+    {
+        testWithDataValidation(false);
+    }
+
+    @Test(timeOut = 30_000)
+    public void testBasicWithDataValidationEnabled()
+            throws Exception
+    {
+        testWithDataValidation(true);
+    }
+
+    @Test(invocationCount = 10)
+    public void testStress()
+            throws ExecutionException, InterruptedException, URISyntaxException, IOException
     {
         CacheConfig cacheConfig = new CacheConfig()
                 .setCacheType(ALLUXIO)
                 .setCachingEnabled(true)
                 .setBaseDirectory(cacheDirectory);
+        AlluxioCacheConfig alluxioCacheConfig = new AlluxioCacheConfig()
```

Comment:
Not really sure what got changed here? If nothing, then can you please fix the format to make sure it does not show in diff?

Reply:
It is just the formatter, I will revert it back to make the diff minimal.